libs/application/templates/driving-license/src/fields/ExamplePaymentPendingField.tsx

```diff
@@ -46,11 +45,33 @@ export const ExamplePaymentPendingField: FC<Props> = ({
       {error && { error }}
 
       {!paymentStatus.fulfilled && (
-        <Box>
-          <Text variant="h2">{m.examplePaymentPendingField}</Text>
-          <Text marginTop="gutter">{m.examplePaymentPendingDescription}</Text>
+        <Box height="full">
+          <Text variant="h3">Augnablik meðan beðið er eftir staðfestingu</Text>
```

Comment:
same as above - 'noticed that you've added this to messages.ts but perhaps forgetting to use here?'

Reply:
Fixed, Thanks.